odkx-src/xlsx-converter-reference.rst

```diff
@@ -388,6 +388,21 @@ The following prompt types are available in ODK-X Survey.
       | like to enter.
   * - video
     - | Used to capture a video.
+  * - Text Area
+    - | Used to allow the user to write upto 255 characters set by default
```

Comment:
This is incorrect, the text area provides a big text area for users to enter information. The note should be if you anticipate users writing anything longer than 255 characters you need to change the model sheet and change the elementType column. As the documentation states: "For example, as shown in the datatypes XLSX, string variables' length can be adjusted from a default of 255 to other lengths with string(len)." https://docs.odk-x.org/xlsx-converter-reference/#model

Reply:
This statement was confusing ...